setuper/setuper.py

```diff
@@ -85,5 +83,33 @@ def setup_account(account_name):
 
     print("-----------------------------------------------")
     print("Account created:", account_name)
-    print("Login at %s with\n\tlogin: %s \n\tpassword: %s" % (SERVER, account_name, account_name))
+    print("Login at %s with\n\tlogin: %s \n\tpassword: %s" % (server_url, account_name, account_name))
     print("-----------------------------------------------")
+    return account_name
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Provide credentials for the setuper")
+    parser.add_argument("-u", "--username", type=str, help="User name")
+    parser.add_argument("-p", "--password", type=str, help="Password")
+    parser.add_argument("-s", "--server_url", type=str, help="Server URL")
+    parser.add_argument("-a", "--additional_projects", action="store_true")
+
+    args = parser.parse_args()
+    server_url = args.server_url
+    username = args.username
+    password = args.password
+    additional_projects = args.additional_projects
+
+    if server_url is None or username is None or password is None:
+        from credentials import *
```

Comment:
Maybe we can catch the case were I call the script, but forget one of the params. Right now I get an error because of the credentials file not existing:

```shell
$ python3 setuper.py --username test
Traceback (most recent call last):
  File "/home/bram/Work/iaso/setuper/setuper.py", line 105, in <module>
    from credentials import *
ModuleNotFoundError: No module named 'credentials'
```

I guess it would be better to display the `sys.exit(` message in this case.